Config.md

```diff
@@ -0,0 +1,138 @@
+# Configuring nx-parallel
+
+`nx-parallel` provides flexible parallel computing capabilities, allowing you to control settings like `backend`, `n_jobs`, `verbose`, and more. This can be done through two configuration systems: `joblib` and `NetworkX`. This guide explains how to configure `nx-parallel` using both systems.
+
+## 1. Using `joblib`
+
+`nx-parallel` relies on [`joblib.Parallel`](https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html) for parallel computing. You can adjust its settings through the [`parallel_config`](https://joblib.readthedocs.io/en/latest/generated/joblib.parallel_config.html) class provided by `joblib`. For more details, check out the official [joblib documentation](https://joblib.readthedocs.io/en/latest/parallel.html).
+
+### 1.1 Usage
+
+```python
+from joblib import parallel_config
+
+# Setting global configs
+parallel_config(n_jobs=3, verbose=50)
+nx.square_clustering(H)
+
+# Setting configs in a context
+with parallel_config(n_jobs=7, verbose=0):
+    nx.square_clustering(H)
+```
+
+Please refer the [official joblib's documentation](https://joblib.readthedocs.io/en/latest/generated/joblib.parallel_config.html) to better understand the config parameters.
+
+Note: Ensure that `nx.config.backends.parallel.active = False` when using `joblib` for configuration, as NetworkX configurations will override `joblib.parallel_config` settings if `active` is `True`.
+
+## 2. Using `networkx`
+
+To use NetworkX’s configuration system in `nx-parallel`, you must set the `active` flag (in `nx.config.backends.parallel`) to `True`.
+
+### 2.1 Configs in NetworkX for backends
+
+When you import NetworkX, it automatically sets default configurations for all installed backends, including `nx-parallel`.
+
+```python
+import networkx as nx
+
+print(nx.config)
+```
+
+Output:
+
+```
+NetworkXConfig(backend_priority=[], backends=Config(parallel=ParallelConfig(active=False, backend='loky', n_jobs=None, verbose=0, temp_folder=None, max_nbytes='1M', mmap_mode='r', prefer=None, require=None, inner_max_num_threads=None, backend_params={})), cache_converted_graphs=True)
```

Comment:
The output in the markdown is hard to read. Is there a better way to show this? Here's one attempt. But there is probably a better way -- don't worry about it too much. We can always tweak it later.
```suggestion
print(nx.config.backends.parallel)
```

Output:

```
ParallelConfig(active=False, backend='loky', n_jobs=None, verbose=0, temp_folder=None, max_nbytes='1M', mmap_mode='r', prefer=None, require=None, inner_max_num_threads=None, backend_params={})
```